librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -3,6 +3,248 @@
 ---
 name: cmd_debug
 commands:
+  - name: db
+    summary: Breakpoints commands
+    subcommands:
+      - name: db
+        summary: List breakpoints / Add breakpoint
+        cname: cmd_debug_db
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+           - RZ_OUTPUT_MODE_STANDARD
+           - RZ_OUTPUT_MODE_RIZIN
+           - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: dbH
+        summary: Add hardware breakpoint
+        cname: cmd_debug_add_hw_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: db-
+        summary: Remove breakpoint
+        cname: cmd_debug_remove_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: db-*
+        summary: Remove all breakpoints
+        cname: cmd_debug_remove_all_bp
+        args: []
+      - name: db.
+        summary: Show breakpoint info at current offset
+        cname: cmd_debug_show_cur_bp
+        args: []
+      - name: dbc
+        summary: Set a command <cmd> to be run when the breakpoint at address <addr> is hit
+        cname: cmd_debug_command_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+          - name: cmd
+            type: RZ_CMD_ARG_TYPE_CMD
+      - name: dbC
+        summary: Run command and continue until <cmd> returns zero
+        cname: cmd_debug_command_bp_continue
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+          - name: cmd
+            type: RZ_CMD_ARG_TYPE_CMD
+      - name: dbd
+        summary: Disable breakpoint
+        cname: cmd_debug_disable_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
```

Comment:
this

Reply:
For `dbd`, `dbe`, `dbs` and a few other commands, I specifically check for `*` in their argument. If they use `dbd *`, I disable all the breakpoints. I feel that this is a good feature, and if there is no other way to do so, then I suggest we keep the option to allow for `*` as a possible argument.

Reply:
One option that we could document in the commands themselves (by using for example the `description` section, see https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs/cmd_zign.yaml#L75 and `za??`) could be to add a `dbq` command (it's enough to add RZ_OUTPUT_MODE_QUIET to the `db` command) and suggest people to use `dbd @@c:dbq`.

What do you think? I feel like this is more in line with our effort to reduce the duplicate things and it would also allow users to approach some more advanced feature of our shell. However... I would really like to hear what you think about this. If you feel this is really too convoluted for users, we can probably keep the `addr` argument.